source/index.html.md

```diff
@@ -14,16 +14,18 @@ includes:
   - users
   - providers
   - products
-  - bulks
   - orders
-  - tags
-  - campaigns
   - invoices
   - issuers
   - payment_methods
-  - payments
-  - address
+  - payments  
   - phone_nr_verification
+  - In development
```

Comment:
does this even work? these ids match a file from the `includes` folder, so I'm not sure how this is working.